src/java.base/share/classes/java/math/BigInteger.java

```diff
@@ -4096,6 +4098,15 @@ public String toString(int radix) {
 
         BigInteger abs = this.abs();
 
+        if (mag.length <= SCHOENHAGE_BASE_CONVERSION_THRESHOLD && radix == 10) {
+            byte[] buf = smallToString(signum < 0, abs);
+            try {
+                return BigDecimal.jla.newStringNoRepl(buf, StandardCharsets.ISO_8859_1);
```

Comment:
The usages of `BigDecimal.jla` and its String computation methods may trigger initialization of BigDecimal when using BigInteger. I recommend moving these string computation utilities and the jla field to a new package-private class within java.math (or a new class  in jdk.internal.math package)